newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -213,3 +225,35 @@ export const onAutoSaveProject = (
 
 export const getWriteErrorMessage = (error: Error): MessageDescriptor =>
   t`An error occurred when saving the project. Please try again by choosing another location.`;
+
+export const onRenderNewProjectSaveAsLocationChooser = ({
+  saveAsLocation,
+  setSaveAsLocation,
+}: {
+  saveAsLocation: ?SaveAsLocation,
+  setSaveAsLocation: (?SaveAsLocation) => void,
+}) => {
+  const outputPath = saveAsLocation
+    ? path.dirname(saveAsLocation.fileIdentifier)
+    : app
+    ? findEmptyPathInDefaultFolder(app)
+    : '';
+  if (!saveAsLocation) {
+    setSaveAsLocation({
+      fileIdentifier: path.join(outputPath, 'game.json'),
```

Comment:
we can't re-use the project name for the filename here?

Reply:
(I don't know if it's something we do elsewhere)

Reply:
We could... for now I'm keeping the existing behavior because I think there are a few safety checks to do for the filename.